tools/cli/workflow.go

```diff
@@ -259,6 +259,14 @@ func newWorkflowCommands() []cli.Command {
 					Name:  FlagResetBadBinaryChecksum,
 					Usage: "Binary checksum for resetType of BadBinary",
 				},
+				cli.StringFlag{
+					Name: FlagEarliestTimeWithAlias,
+					Usage: "EarliestTime of decision start time, for resetType of DecisionStartedTime." +
+						"Supported formats are '2006-01-02T15:04:05+07:00', raw UnixNano and " +
+						"time range (N<duration>), where 0 < N < 1000000 and duration (full-notation/short-notation) can be second/s, " +
+						"minute/m, hour/h, day/d, week/w, month/M or year/y. For example, '15minute' or '15m' implies last 15 minutes, " +
+						"meaning that workflow will be reset to the first decision that started in last 15 minutes.",
```

Comment:
Is this a required option for DecisionStartedTime? It seems like if this flag is not set, this type is the same as FristDecisionCompleted？Shall we make it more clear in the comments?

Reply:
Yes this is a required option. 
https://github.com/uber/cadence/blob/a9337097fb6410818ccf02be99faf3f2b27a3f14/tools/cli/workflowCommands.go#L1449

https://github.com/uber/cadence/blob/a9337097fb6410818ccf02be99faf3f2b27a3f14/tools/cli/workflowCommands.go#L1540

We will check the extra option is existing for the particular resetType